README.md

```diff
@@ -135,6 +135,25 @@ To use salt-lint with [pre-commit](https://pre-commit.com),  just add the follow
 
 # Rules
 
+
+## List of rules
+
+| Rule                                                 | Description                                                                  |
+|:-----------------------------------------------------|:-----------------------------------------------------------------------------|
```

Comment:
3-symbols is actually enough for each column:

```suggestion
|:-:|:--|
```

However, the first and last bars are redundant, so you could simplify that to:

```suggestion
:-:|:--
```

And then remove the leading and trailing bars from each line in the table.